itest/lnd_onchain_test.go

```diff
@@ -466,7 +466,8 @@ func testAnchorThirdPartySpend(ht *lntest.HarnessTest) {
 	// fully.
 	aliceChanPoint1 := ht.OpenChannel(
 		alice, bob, lntest.OpenChannelParams{
-			Amt: firstChanSize,
+			Amt:  firstChanSize,
+			Memo: "bob is a good peer",
```

Comment:
nit: make the memo a const?

Reply:
Yes a constant `testMemo` would shorten quite a few things in the itest.

Reply:
Good idea, updated :slightly_smiling_face: